bibtex-actions.el

```diff
@@ -41,10 +41,14 @@
 
 (declare-function org-element-context "org-element")
 (declare-function org-element-property "org-element")
+(declare-function org-element-type "org-element")
+(declare-function org-cite-get-references "org-cite")
```

Comment:
If this function doesn't exist (because user is using an older version of org), that's no problem; right?

Reply:
Yes. `bibtex-actions-get-key-org-cite` just returns nil if `org-cite` is not available.